lib/parse_statement.rb

```diff
@@ -18,12 +21,12 @@ def self.parse(file)
     end
 
     attr_reader :raw
-    statement_attr :type,    'TRNTYPE'
-    statement_attr(:id,      'FITID', &:to_i)
-    statement_attr :name,    'NAME'
-    statement_attr :memo,    'MEMO'
-    statement_attr(:date,    'DTPOSTED'){ |string| DateTime.parse(string) }
-    statement_attr(:amount, 'TRNAMT'){ |string| BigDecimal.new(string) }
+    statement_attr(:type,   "TRNTYPE")
+    statement_attr(:id,     "FITID", &:to_i)
+    statement_attr(:name,   "NAME")
+    statement_attr(:memo,   "MEMO")
+    statement_attr(:date,   "DTPOSTED") { |string| DateTime.parse(string) }
```

Comment:
Do not use `DateTime.parse` without zone. Use one of `Time.zone.parse`, `DateTime.parse.current`, `DateTime.parse.in_time_zone`, `DateTime.parse.utc`, `DateTime.parse.getlocal`, `DateTime.parse.iso8601`, `DateTime.parse.jisx0301`, `DateTime.parse.rfc3339`, `DateTime.parse.to_i`, `DateTime.parse.to_f` instead.